index.js

```diff
@@ -126,10 +128,19 @@ nativeBindings.nativeGl.onconstruct = (gl, canvas) => {
 
     const document = canvas.ownerDocument;
     const window = document.defaultView;
-    const framebufferWidth = nativeWindow.getFramebufferSize(windowHandle).width;
-    window.devicePixelRatio = framebufferWidth / canvasWidth;
 
-    const title = `Exokit ${version}`
+    const nativeWindowSize = nativeWindow.getFramebufferSize(windowHandle);
+    const nativeWindowHeight = nativeWindowSize.height;
+    const nativeWindowWidth = nativeWindowSize.width;
+
+    // Calculate devicePixelRatio.
+    window.devicePixelRatio = nativeWindowWidth / canvasWidth;
+
+    // Tell DOM how large the window is.
+    canvas.height = window.innerHeight = nativeWindowHeight / window.devicePixelRatio;
```

Comment:
Also, I think setting `canvas.{width,height}` might be doing nothing here besides wasting CPU recreating context, because the `canvas` already had this DOM metric size.

Reply:
I can fix this up.